test/profile/both.def.exp2

```diff
@@ -0,0 +1,5 @@
+
+FUNCTIONS
+	_Dmain
```

Comment:
I know - I was just FYIing the CI error.
I didn't add the whitespace check here, but it seems that [`.exp` files are explicitly filtered out](https://github.com/dlang/druntime/blob/master/posix.mak#L274). So the simple fix is to rename the file ...